packages/slider/src/presenters/SliderPresenter.js

```diff
@@ -40,7 +67,28 @@ export default class SliderPresenter extends Component {
             resolvedRoles
           );
 
-          return (
+          const marks =
+            type === sliderTypes.DISCRETE
+              ? this.getTickMarks(min, max, otherProps.step, styles)
+              : null;
+
+          return marks ? (
+            <div style={{ position: "relative" }}>
```

Comment:
Instead of having a random `style` prop here, we should get this from `stylesheet.js`, this makes it easier for folks to override when the `stylesheet` prop is implemented as well.